client/src/components/Map.jsx

```diff
@@ -48,7 +50,51 @@ const Map = () => {
       mapboxApiAccessToken={process.env.REACT_APP_MAPBOX_API_KEY}
       dragRotate={false}
       touchRotate={false}
-    />
+    >
+      <div
+        className="searchBar"
+        style={{ position: "absolute", left: 20, top: 20 }}
+      >
+        <input
+          class="focus:outline-none focus:shadow-outline pl-2 mr-0.75 rounded-sm h-10 border-2 w-64"
+          type="search"
+          placeholder="Search for a location..."
+          aria-label="Search for a location"
+        />
+        <button
+          class="bg-white rounded-sm p-2 focus:outline-none focus:shadow-outline h-10 leading-tight border-t-2 border-b-2 border-r-2"
+          type="submit"
+          aria-label="Submit Search"
+        >
+          icon
+        </button>
+      </div>
+      <div
+        className="legend"
+        class="bg-white p-1 rounded-sm"
+        style={{ position: "absolute", right: 40, top: 20 }}
+      >
+        <details>
+          <summary class="p-2">
+            <img
+              class="w-5 h-5 inline mr-1 mb-1"
+              src="information.png"
+              alt="i"
+            />
+            Legend
+          </summary>
+          <div class="w-64">
+            <img src="legend.png" alt="legend" />
```

Comment:
probably something more descriptive is useful here -- for example, we would want to say `alt="Individual Arrests 0 to 30+"`